commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/ext/JsonParserExt.kt

```diff
@@ -0,0 +1,75 @@
+package ac.uk.ebi.biostd.ext
+
+import ac.uk.ebi.biostd.common.deserialization.stream.StreamDeserializerBuilder
+import com.fasterxml.jackson.core.JsonParseException
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.core.JsonToken
+
+/**
```

Comment:
I would expect a single method with the call to `nexToken` which is the iterator one.